devel/cmake-devel/Portfile

```diff
@@ -80,13 +80,13 @@ patchfiles-append \
     patch-remove-homebrew-references.diff
 
 depends_lib-append \
+    path:lib/libuv.dylib:libuv \
```

Comment:
There has previously been a complaint that using dylib paths in dependencies makes ports incompatible with non-Mac operating systems since they don't use dylibs. Can you please change this to `path:lib/pkgconfig/libuv.pc:libuv` so that it is platform neutral?

Reply:
A fair point, yeah.